vendor/github.com/go-yaml/yaml/resolve.go

```diff
@@ -180,7 +180,7 @@ func resolve(tag string, in string) (rtag string, out interface{}) {
 					return yaml_INT_TAG, uintv
 				}
 			} else if strings.HasPrefix(plain, "-0b") {
-				intv, err := strconv.ParseInt("-" + plain[3:], 2, 64)
+				intv, err := strconv.ParseInt("-"+plain[3:], 2, 64)
```

Comment:
Another nit, some string concats have spaces, others don't

Reply:
This is within the vendor file... no control over it.... Guessing when I ran `go-fmt` it pulled in the stuff from vendor

Reply:
oh, sorry, i didn't notice the `/vendor` path